plugins/inputs/ipvs/README.md

```diff
@@ -0,0 +1,25 @@
+# IPVS Input Plugin (Linux)
+
+The IPVS input plugin uses the linux kernel netlink socket interface to gather
+metrics about ipvs virtual and real servers.
+
+## Configuration
+
+None
```

Comment:
Add this as a ```toml block, it is the output of `./telegraf -usage ipvs` without the plugin description.
```
[[inputs.ipvs]]
  # no configuration
```

Reply:
```suggestion
[[inputs.ipvs]]
  # no configuration
```